KnowledgeBase.Data/Repositories/ResourceRepository.cs

```diff
@@ -0,0 +1,18 @@
+﻿using KnowledgeBase.Data.Data;
+using KnowledgeBase.Data.Models;
+using KnowledgeBase.Data.Repositories.Interfaces;
+
+namespace KnowledgeBase.Data.Repositories;
+
+public class ResourceRepository : GenericRepository<Resource>, IGenericRepository<Resource>, IResourceRepository
+{
+	public ResourceRepository(KnowledgeDbContext context) : base(context)
+	{
+	}
+
+	public void IsRemoved(Resource resource)
```

Comment:
`IsRemoved` sounds like a question. If you want to have method that deletes resource just call it `Delete`